test_mpop_service/TestMpopService.cpp

```diff
@@ -24,7 +24,7 @@ void TestMpopService::removeDatabaseTestEntries() {
 
     if (this->is_mysql_supported) {
         // Call deleteTagsVisitorsAndTheirAnswers instead of deleteAllFromDatabase, so that we can run these tests in prod.
-        this->facade->deleteAllFromDatabase();
+        //this->facade->deleteAllFromDatabase();
```

Comment:
We should call `deleteTagsVisitorsAndTheirAnswers(const QList<QString>& rfidTags)` here.